tests/integ/modin/frame/test_loc.py

```diff
@@ -464,7 +466,7 @@ def test_mi_df_loc_get_non_boolean_list_col_key(mi_table_df, key, native_error):
         if native_error:
             with pytest.raises(native_error):
                 _ = mi_table_df.loc[:, key]
-            assert df.loc[:, key].empty
+                assert df.loc[:, key].empty
```

Comment:
```suggestion
```

The previous line should raise the error, so I don't think we reach this assertion.